index.d.ts

```diff
@@ -0,0 +1,50 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string. If the variable is
+   * not set and a default was specified, the default value is returned,
+   * otherwise an error is thrown.
+   */
```

Comment:
I see. Since there's a practical benefit I'm happy to set aside my aesthetic concerns. :)